lib/property_sets/active_record_extension.rb

```diff
@@ -184,6 +184,38 @@ def update_property_set_attributes(attributes)
           end
         end
       end
+
+      def update_columns(attributes)
+        attrs = if delegated_property_sets?
+          attributes.reject{|k,_| self.class.delegated_property_set_attributes.include?(k.to_s) }
+        else
+          attributes
+        end
+
+        raise ArgumentError, "Empty list of attributes to change" if attrs.empty?
```

Comment:
if `super` does not do this, then we should not do it either ?

Reply:
It does in certain versions of rails. In later versions, rails just lets it blow up. Do we have a preference?

Reply:
do what rails does 🤷 
... not property_set gem responsibility to unify rails apis